src/components/ChatComponents/ChatControls.tsx

```diff
@@ -106,36 +98,25 @@ const ChatControls: React.FC<ChatControlsProps> = ({
 
   return (
     <div className="chat-icons-container">
-      <button className="chat-icon-button clickable-icon" onClick={() => onNewChat(false)}>
-        <RefreshIcon className="icon-scaler" />
-        <span className="tooltip-text">
-          New Chat
-          <br />
-          (unsaved history will be lost)
-        </span>
-      </button>
-      <button className="chat-icon-button clickable-icon" onClick={onSaveAsNote}>
-        <SaveAsNoteIcon className="icon-scaler" />
-        <span className="tooltip-text">Save as Note</span>
-      </button>
+      <TooltipActionButton
+        onClick={() => {
+          onNewChat(false);
+        }}
+        Icon={<RefreshIcon className="icon-scaler" />}
+      >
+        <div>New Chat</div>
+        {!settings.autosaveChat && <div>(Unsaved history will be lost)</div>}
+      </TooltipActionButton>
+      <TooltipActionButton onClick={onSaveAsNote} Icon={<SaveAsNoteIcon className="icon-scaler" />}>
+        Save as Note
+      </TooltipActionButton>
       {selectedChain === "vault_qa" && (
-        <>
-          <button className="chat-icon-button clickable-icon" onClick={onRefreshVaultContext}>
-            <UseActiveNoteAsContextIcon className="icon-scaler" />
-            <div
-              className="tooltip-text"
-              style={{
-                transform: "translateX(-90%)",
-              }}
-            >
-              <span>Refresh Index for Vault</span>
-            </div>
-          </button>
-          {/* <button className="chat-icon-button clickable-icon" onClick={handleFindSimilarNotes}>
-            <ConnectionIcon className="icon-scaler" />
-            <span className="tooltip-text">Find Similar Notes for Active Note</span>
-          </button> */}
-        </>
+        <TooltipActionButton
+          onClick={onSaveAsNote}
+          Icon={<UseActiveNoteAsContextIcon className="icon-scaler" />}
```

Comment:
Since we are updating this, `UseActiveNoteAsContextIcon` is a legacy name, we can rename it to "RefreshIndexIcon" or something.

Reply:
I saw we are using lucide icons. I updated them to use the library instead of custom Icons. WDYT?

Reply:
we can follow up to replace the rest of icons in the app.

Reply:
@zeroliu That would be great! These were added by a total frontend noob - me. Glad this will be cleaned up now!